api/lib/mno_enterprise/concerns/controllers/pages_controller.rb

```diff
@@ -59,9 +59,16 @@ def app_logout
   def terms
     @meta[:title] = 'Terms of Use'
     @meta[:description] = 'Terms of Use'
-    ts = MnoEnterprise::App.order_by("updated_at.desc").first.updated_at
-    @apps = Rails.cache.fetch(['pages/terms/app-list', ts]) do
-      MnoEnterprise::App.order_by("name.ac").reject{|i| i.terms_url.blank?}
+
+    ts = MnoEnterprise::App.order_by("updated_at.desc").first
```

Comment:
Ah ok I replaced it with the try method. I was trying to run that line of code in my terminal but was using irb/pry instead of rails console. Didn't realize the method was coming from rails. 👍